core/sdk-core/src/main/java/software/amazon/awssdk/core/async/AsyncRequestBody.java

```diff
@@ -50,7 +50,6 @@
  * @see FileAsyncRequestBody
  * @see ByteArrayAsyncRequestBody
  */
-@ReviewBeforeRelease("This is exactly the same of SdkHttpRequestProvider. Can we just have one?")
```

Comment:
Did you verify that we can't combine both classes? I has this task last week but couldn't get to it

Reply:
Not without a breaking change.

Reply:
Do you think its useful to have both? Will there be any duplications in future and cause operational pain? If so, I'd vote for breaking change and merge them.